### ISSUE TITLE
Kudos can't be taken away.

### ISSUE THREAD

**Issue by jonathanhadams** [themeskult/wp-svbtle#58]
Refreshing the page after adding Kudos prevents Kudos to be removed, or added again.

**Comment by dainiuxt** [comment]
For me as well.

**Comment by jmergy** [comment]
I went in to the theme editor and removed it >> http://mergy.org

**Comment by dainiuxt** [comment]
I would like to leave functionality, but sometimes I accidently hover on Kudos and it counts, and after that it is unavailable to remove once you "Kudos".

**Comment by jmergy** [comment]
I never got it to actually "stick" no matter what I did, so always having 0 was a big bummer.

**Comment by dainiuxt** [comment]
Actually, I think this is not a bug. This is philosophy. Should be closed.

**Comment by rickyrauch** [comment]
Open style.css and add this line at the bottom:

aside.kudo{display:none !important}

:)

**Comment by criptych** [comment]
> This is philosophy.

This is propaganda.  I agree with @jaepage:  new users aren't told what they're getting into, and they aren't allowed to back out.  The only indication I got that the widget did anything but display a counter (and the entire reason I researched the problem and found this issue) was the message "Don't move" when I hovered over it -- didn't even _click_, just hovered -- and the counter going up.  After which hovering again, clicking, etc. did nothing.  I didn't even have to refresh the page like OP mentions.

A good user interface doesn't punish exploration ([Schneiderman's Golden Rule #6](https://faculty.washington.edu/jtenenbg/courses/360/f04/sessions/schneidermanGoldenRules.html)).
